docs/installing.rst

```diff
@@ -1,12 +0,0 @@
-.. index:: installing
```

Comment:
Why delete this?

Reply:
I am not sure I understand. I believe the entire file was deleted because it didn't really convey anything or do anything - it was a duplicate with something else. 

I think though I will have to go over the organization of this part because depending on how things are accessed, stuff can look funny.